ng/src/web/entity/page.js

```diff
@@ -179,15 +185,15 @@ class EntityPage extends React.Component {
     }
 
     return (
-      <Layout flex="column" grow="1">
+      <StyledLayout flex="column" grow="1">
```

Comment:
Just set align="start" see https://github.com/greenbone/gsa/blob/master/ng/src/web/components/layout/withLayout.js instead of creating a new component. If you are using Layout don't override flex layout based css by creating a new parent component.